btrdbextras/eventproc/protobuff/api_pb2_grpc.py.bak

```diff
@@ -0,0 +1,165 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
```

Comment:
Do we want to add the .bak file?

Reply:
you mean the .bak file referenced in Makefile?  I would not want to include that particular file but I feel like you are referring to something else?

Reply:
ack, sorry must have been looking at something else
removing this file!